models/model1_part2.ipynb

```diff
@@ -0,0 +1,611 @@
+{
```

Comment:
<p>If this code will be reused in multiple notebooks, I recommend moving it to a <code>utils/text_preprocessing.py</code> file.</p><p>Same code is in the other PR </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
We are thinking of moving both the notebooks of the first model into a single one. We are not sure if we will use the same code for the coming models but if we do, we will make sure to include the redundant code in a utils folder like you have suggested.